fw.lua

```diff
@@ -321,7 +327,7 @@ local function _log_event(self, request_client, request_uri, rule, match)
 				}
 			end
 
-			socket_logger.log(cjson_encode(t) .. "\n")
+			socket_logger.log(cjson.encode(t) .. "\n")
```

Comment:
This is fixed in a separate commit (26a27e6686a4a297a4193ddae5ab2414bc9e408f). Please keep commits atomic.